src/child/actions/window.js

```diff
@@ -53,6 +58,59 @@ const updatingOptionsError = createActionCreator(() => ({
     type: ACTION_TYPES.UPDATING_OPTIONS_ERROR
 }));
 
+export const resizeWindow = createActionCreator((dimensions) => ({
+    type: ACTION_TYPES.RESIZE_WINDOW,
+    previousExpandedDimensions: dimensions
+}));
+
+export function resizeToMin() {
```

Comment:
Why `resizeTo[X]` (for `Min`, `Max` and `Restore`)? They're no calling `resizeTo` (as far as I'm aware?), so would `minimize`, `maximize` and `restore` be more appropriate?